libraries/stencil/.gitignore

```diff
@@ -0,0 +1,26 @@
+dist/
```

Comment:
I don't think any of the other libraries include their own `.gitignore`. Is this needed?

Reply:
Since Stencil is a compiler we do need to ignore the ouput/cache folders or it will bloat the repository, however I cleaned the gitignore so its just stencil specific stuff is that okay?